contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -158,11 +76,22 @@
         type: Array,
         default: () => [],
       },
+      /**
+       * inputComponent is a component that will be used to render the input(s)
+       * that will be used to edit the boolean map field.
+       * It should be a component that accepts a v-model prop and a nodeIds prop.
+       *
+       * The v-model prop should support an object with the same structure as the
+       * `this.selectedValues` data property.
+       */
+      inputComponent: {
```

Comment:
In general, passing a component as a prop feels like a code smell - we should define a [slot interface](https://v2.vuejs.org/v2/guide/components-slots.html) for this in the template instead, and it should work just as well.

Reply:
I think you're right - I don't know that it necessarily makes sense to have that check box below the search box here. Could we check in with @jtamiace to see if there would be any issues with moving it above?

Reply:
I checked with Jessica and she told me that there was no problem with moving it above :open_hands: